release/nightly_tests/placement_group_tests/placement_group_performance_test.py

```diff
@@ -0,0 +1,158 @@
+import argparse
+import json
+import os
+
+import ray
+from ray._private.ray_microbenchmark_helpers import timeit
+
+RESOURCES_VALUE = 0.01
+
+
+def test_placement_group_perf(num_pgs, num_bundles, num_pending_pgs):
+    # Run the placement group performance benchmark given arguments.
+    ray.cluster_resources()["custom"] >= (
+        RESOURCES_VALUE * num_pgs * num_bundles)
```

Comment:
Do we need a while loop? Or do we need assert ?

Reply:
oh I didn't realize I don' have assert. We need assert (when the code is running, all resouces should be prepared)